Makefile

```diff
@@ -0,0 +1,2 @@
+build:
+	parcel build packages/*/*/
```

Comment:
This is a bit of a hack, but if you put this in the package script, Yarn borks because it can't deal with the glob pattern. Unsure why.

Reply:
That said, I think make is kind of nice for composing scripts in a more readable format than all of the package.json noise. I wouldn't complain if we decided to use it more 🙂

Reply:
I've just had a look at this and it's zsh that throws on zero-match glob patterns. The trick is to quote the glob so that it's actually **parcel** that expands the glob instead of your shell.

```jsx
"build": "parcel build 'packages/*/*/'",
```